### PR TITLE
update eslint config

### PR DESCRIPTION
**Pull request by sam-github**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,11 +25,11 @@
   },
   "homepage": "https://github.com/strongloop/express-example-app",
   "dependencies": {
+    "eslint-config-strongloop": "^2.1.0",
```
Comment (rmg) [inline package.json]: Should be in `devDependencies`, shouldn't it?

File: index.js [hunk 2]
```diff
@@ -13,8 +13,11 @@ module.exports = function() {
 
   app.get('/', function(req, res) {
     var s = req.socket;
-    var msg = fmt('Hello, World! Request served from %s:%d\n',
-      s.localAddress, s.localPort);
+    var msg = fmt(
```
Comment (rmg) [inline index.js]: Did `eslint` do this?

**Review by rmg — CHANGES_REQUESTED** [review]

File: index.js (hunk 2 above)
Comment (sam-github) [reply index.js]: I thought the eslint rules required it, but just checked, and it looks like it didn't. Not sure what happened, this was something I did a month ago. Anyhow, backed it out.

**Review by rmg — APPROVED** [review]
LGTM :shipit:

**Comment by rmg** [comment]
Maybe set the engine in `package.json` since this will no longer "build" on node < 4.x

**Pull request merged**